contracts/OysterPearl.sol

```diff
@@ -13,32 +13,33 @@ contract OysterPearl {
   bool public saleClosed;
   bool public directorLock;
   uint256 public claimAmount;
-  uint256 public payAmount;
-  uint256 public feeAmount;
+  uint256 public payPercentage:
+  uint256 public feePercentage;
   uint256 public epoch;
   uint256 public retentionMax;
 
   // Array definitions
+  mapping (bytes32 => uint256) public hashBalances;
   mapping (address => uint256) public balances;
   mapping (address => mapping (address => uint256)) public allowance;
-  mapping (address => bool) public buried;
-  mapping (address => uint256) public claimed;
+  mapping (bytes32 => bool) public buried;
```

Comment:
Is there a reason why we switched from type `address` to `byte32`?

Reply:
Keccak256 returns bytes32